styles/01-evening.json

```diff
@@ -119,7 +119,7 @@
 			}
 		},
 		"variations": {
-			"section-1": {
```

Comment:
Do we want to order the sections by number in the different style variations?

Reply:
Oh I did not think of that. I will make an update in the afternoon.

Reply:
I did not find the time yesterday, trying it now.